src/crate/qa/tests.py

```diff
@@ -223,7 +224,7 @@ def new_node(version, settings={}):
             n._settings = s  # CrateNode does not hold its settings
             self._add_log_consumer(n)
             self._on_stop.append(n)
-            return n
+            return (n, version_tuple)
```

Comment:
@chaudum maybe you have a better suggestion

Reply:
or @mfussenegger ?

Reply:
looks fine to me